lib/solr_wrapper/instance.rb

```diff
@@ -101,37 +101,41 @@ def restart
     end
 
     ##
-    # Check the status of a managed Solr service
-    def status
-      return true unless config.managed?
+    # Is Solr running?
+    def started?
+      status_info =~ /running on port #{port}/ && true || false
```

Comment:
I commented on this previously. Please remove `&& true || false`. It should be sufficient that `started?` return a truthy value. If that isn't the case, let's be explicit about that using `match()` and checking for a return value.